src/Price/index.tsx

```diff
@@ -53,12 +59,19 @@ const Price: FunctionComponent<Props> = ({
   )
 
   let productDiscount
+  let productDiscountPercentage
   if (discountPrice) {
-    productDiscount = <p className={styles['price-cut']}>{formatPrice(price)}</p>
+    if (discountPercentage) {
+      let priceDiff = price[0]-discountPrice[0];
+      let percentage = Math.round((priceDiff/price[0])*100);
+      productDiscount = <p className={styles['price-cut']}><span className={clsx({[styles['price-cut-percent']]: percentage > 0})}>{percentage > 0 ? percentage + `% Off ` : ``}</span><span className={styles['price-cut-value']}>{formatPrice(price)}</span></p>
```

Comment:
do you have prettier installed? This should have auto-formatted it...unless you have windows. I'll pull down your branch and run prettier and try to fix the windows thing.